setup/0.init.sh

```diff
@@ -135,3 +135,6 @@ read -p "Press enter to continue" answer
 # ╚═╝╚═╝  ╚═══╝╚══════╝   ╚═╝   ╚═╝  ╚═╝╚══════╝╚══════╝     ╚═════╝ ╚═╝  ╚═╝╚═╝  ╚═══╝╚═╝  ╚═╝ ╚═════╝╚═╝  ╚═╝╚══════╝
                                                                                                                      
 . ./setup/4.install_ganache.sh
+
+echo 'finally, you will need to configure your hosts file to make kong proxy work correctly.'
+echo 'you can used the script /scripts/configure-hosts.sh to do so.'
```

Comment:
We are not running this and pushing this onto the user because it is the hosts file, right? We should explicitly state that we aren't going to run this for them for that reason - might be confusing to a user because we just set up everything else for them.